gluoncv/model_zoo/mask_rcnn/mask_rcnn.py

```diff
@@ -158,21 +169,23 @@ class MaskRCNN(FasterRCNN):
         Names of categories, its length is ``num_class``.
     mask_channels : int, default is 256
         Number of channels in mask prediction
-    deep_fcn : boolean, default False
-            Whether to use deep mask branch (4 convs)
+    rcnn_max_dets : int, default is 1000
```

Comment:
I think this one need a guard as well, I remember @hetong007  had a problem when setting `rpn_test_post_nms` to small number and it failed. Probably we can add it to unittest?

Reply:
done